baselines/fedmix/README.md

```diff
@@ -0,0 +1,128 @@
+---
+title: FedMix Approximation of Mixup under Mean Augmented Federated Learning
+url: https://arxiv.org/abs/2107.00233
+labels: ["data heterogeneity", "mixup", "non-iid"]
+dataset: ["cifar10", "cifar100", "femnist"]
+---
+
+# FedMix: Approximation of Mixup under Mean Augmented Federated Learning
+
+****Paper:**** [arxiv.org/abs/2107.00233](https://arxiv.org/abs/2107.00233)
```

Comment:
```suggestion
**Paper:** [arxiv.org/abs/2107.00233](https://arxiv.org/abs/2107.00233)
```

Reply:
In many places in the readme `****` is used instead of `**`. Could you correct these?